Source/s_sound.c

```diff
@@ -759,6 +759,13 @@ void S_Start(void)
             mnum = spmus[gamemap-1];
       }
    }
+   else
+   {
+      if (gamemode == commercial)
```

Comment:
I prefer the Pr+ approach of wrapping music indices around. Else e.g. MAP33 will play the "wrong" music.